randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1560,6 +1567,10 @@ Extra - asset_id: collision_camera_024
   * Pickup 112; Major Location? True
   * Extra - actor_name: item_energyfragment
   * Extra - actor_def: actordef:actors/items/item_energyfragment/charclasses/item_energyfragment.bmsad
+  > Door to Above Pulse Radar
+      All of the following:
+          Pseudo-Wave Beam (Intermediate)
+          Lay Bomb or Lay Cross Comb or Lay Power Bomb or Shoot Charge Beam
```

Comment:
Probably shouldn't include this right now, since logic will assume you can reach the door immediately from that node. Especially the last combo is impossible: Pseudo and Charge Beam. That's not possible to traverse without some way to use the tunnel. Best to remove and make an issue for it or something.